scripts/helm-nsm-install.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# A script for wrapping NSM installation via Helm. Both Helm 2 and Helm 3 are
+# supported.
+
+function with_helm_2 () {
+  # We specifically set admission-webhook variables here as it is a subchart
+  # there might be a way to set these as global and refer to them with .Values.global.org
+  # but that seems more intrusive than this hack. Consider changing to global if the charts
+  # get even more complicated
+  helm install --name="$CHART" \
+  --atomic --timeout 300 \
+  --set org="$CONTAINER_REPO",tag="$CONTAINER_TAG" \
+  --set forwardingPlane="$FORWARDING_PLANE" \
+  --set insecure="$INSECURE" \
+  --set global.JaegerTracing="true" \
+  --set spire.enabled="$SPIRE_ENABLED",spire.org="$CONTAINER_REPO",spire.tag="$CONTAINER_TAG" \
+  --set admission-webhook.org="$CONTAINER_REPO",admission-webhook.tag="$CONTAINER_TAG" \
+  --set prefix-service.org="$CONTAINER_REPO",prefix-service.tag="$CONTAINER_TAG" \
+  --namespace="$NSM_NAMESPACE" \
+  deployments/helm/"$CHART"
+
+}
+
+function with_helm_3 () {
+  # Ensure the nsm namespace has been created
+  kubectl config use-context kind-nsm
```

Comment:
Please, do not refer to kind in this script. Using kind is one way to deploy NSM, but for development some may use Vagrant or others. Keep the script more generic.

Reply:
Ack, we can probably do something different here.